db/migrate/20210831143624_add_get_applications_indexes_to_application_choice.rb

```diff
@@ -0,0 +1,8 @@
+class AddGetApplicationsIndexesToApplicationChoice < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :application_choices, :provider_ids, using: 'gin', if_not_exists: true, algorithm: :concurrently
```

Comment:
I was having a look at an existing long index migration we had to do: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/db/migrate/20210319163050_add_auditable_type_id_index_to_audits.rb#L7

Do we need to do something similar with setting the `statement_timeout` to 0 to avoid issues as you mentioned this will take 10 minutes and this will prob cause a timeout?

Reply:
Good point, updated

Reply:
Added `safety_assured` to appease `strong_migrations` and tested that the migration can be reversed.